src/BookResourceStore.ts

```diff
@@ -0,0 +1,97 @@
+import Manifest from "Manifest";
+
+export interface IBookStore {
+  href: string;
+  data: Blob;
+}
+
+// IndexDB store for Book resources
+// Only works on browsers that support IndexedDB.
+export default class BookResourceStore {
+  private readonly db: IDBDatabase;
+
+  public constructor(db: IDBDatabase) {
+    this.db = db;
+  }
+
+  public static createBookResourceStore(): Promise<BookResourceStore> {
+    if (!("indexedDB" in window)) {
+      //TODO: a polyfill to handle lack of IndexedDB
+      console.log("This browser doesn't support IndexedDB");
```

Comment:
There is nothing yet to handle what happens when indexedDB doesn't exist ... 
I filed https://jira.nypl.org/browse/SIMPLY-2914 but it might be good to just have it always load from the remote